Client/src/Validation/validation.js

```
@@ -128,10 +128,43 @@ const maintenanceWindowValidation = joi.object({
+		.trim()
+		.messages({
+			"string.empty": "JWT TTL is required."			
+		}),
+	dbType: joi.string().trim().messages({
+		"string.empty": "DB type is required",
+	}),
+	redisHost: joi.string().trim().messages({
+		"string.empty": "Redis host is required",
+	}),
+	redisPort: joi.string().allow('').custom((value, helpers) => {
+		if(value && isNaN(parseInt(value))){
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)